assets/javascripts/birdeck.js

```diff
@@ -17,32 +22,34 @@ function renderPost(post) {
     + "</p>"
     + "<button id='delete-post' name='button-fetch' class='btn btn-default btn-xs'>Delete</button>"
     + "</div>"
-    )
+  )
+}
+
+function renderPosts(posts) {
+  return posts.map(renderPost)
+}
+
+function addPostToPage(post) {
+  $latestPosts.append(post);
+}
+
+function addPostsToPage(posts) {
+  $latestPosts.html(posts)
+}
+
+function handleError(xhr) {
+  console.log(xhr.responseText)
 }
 
 function fetchPostsButton() {
-  $("#button-fetch-posts").on("click", function(){
-    fetchPosts()
-    })
+  $("#button-fetch-posts").on("click", fetchPosts)
 }
 
 function fetchPosts() {
-  var newestItemID = parseInt($(".post").last().attr("data-id"))
-
-  $.ajax({
-    type:    "GET",
-    url:     "https://turing-birdie.herokuapp.com/api/v1/posts.json",
-    success: function(posts) {
-      $.each(posts, function(index, post) {
-        if (isNaN(newestItemID) || post.id > newestItemID) {
-          renderPost(post)
-        }
-      })
-    },
-    error: function(xhr) {
-      console.log(xhr.responseText)
-    }
-  })
+  $.getJSON("https://turing-birdie.herokuapp.com/api/v1/posts.json")
```

Comment:
@stevekinney I rather have them using the lower level `$.ajax()` since then they can learn the other patterns and understand what's going on underneath it. Also, it clicks with them more since it's similar to other libraries that send requests in Ruby.